README.md

```diff
@@ -16,4 +16,8 @@ You can read the [guide](./docs/README.md) to get started with TizenBrew.
 
 # Making a module
 
-You can read the [guide](./docs/MODULES.md) to make a module for TizenBrew.
\ No newline at end of file
+You can read the [guide](./docs/MODULES.md) to make a module for TizenBrew.
+
+# Translations
+
+You can read the [guide](./docs/TRANSLATION.md) to translate TizenBrew UI language into your native language.
```

Comment:
It'd be better to remeve "UI language" from here.

Reply:
Updated.